percy-mirror-verifier/cloud_build.yaml

```diff
@@ -0,0 +1,41 @@
+steps:
+  # The first two steps are required to identify and inject un-redacted secrets.
+  - name: gcr.io/cloud-builders/npm
+    args:
+      - install
+      - yaml
+  - name: gcr.io/cloud-builders/npm
+    args:
+      - run
+      - replace-secrets
+      - --
+      - percy-mirror-verifier
+    secretEnv:
+      - WEBHOOK_SECRET
+      - GITHUB_CLIENT_SECRET
+      - PRIVATE_KEY
+
+  # Install app dependencies
+  - name: gcr.io/cloud-builders/npm
+    dir: percy-mirror-verifier
+    args:
+      - install
+
+  # Deploy the app.
+  - name: gcr.io/cloud-builders/gcloud
+    dir: percy-mirror-verifier
+    args:
+      - app
+      - deploy
+
+# Overall cloud build timeout
+timeout: 1800s
+
+# These secrets are the base64-encoded form of encrypted secret values. They are
+# automatically decrypted and added to the `.env` environment at build time.
+secrets:
+  - kmsKeyName: projects/amp-percy-mirror-verifier/locations/global/keyRings/amp-github-apps-keyring/cryptoKeys/app-env-key
+    secretEnv:
+      PERCY_WEBHOOK_SECRET: CiQAAQqTNfjJpnLWjUKNGKIewyfQGfwnLbDSAVBaYETiCtzLfzUSSQC1dVigimKVPJUjvqpyepX9y2C1U5p5Cpp/NkkKhReIvlR7I54rzef2MlYqDPGYHQzZozHgv1aVo1ETdp/FlatIx2fvZVDhos0=
```

Comment:
These secrets don't look like placeholders...?

Reply:
These are the encrypted values. Our GCP service account has the private key to decrypt these and it does that during the cloud build phase. See https://github.com/ampproject/amp-github-apps/blob/main/DEPLOYMENT.md for context (shout-out to @rcebulko for this doc... I didn't need to ping him even once when setting this up!)